sensor_api/perfectfit/api/sensor.py

```diff
@@ -0,0 +1,37 @@
+from flask import make_response, request
+
+def http_response(sensor_type):
+    """HTTP responce to a POST request.
+
+    Args:
+        sensor_type(str): "Accelerometer".
+
+    Returns:
+        HTTP responce: Responce to POST request.
+    """
+    req_data = request.data
+    req_data = req_data.decode('utf-8')
+    # TODO: valid userID
+    # valid, message = validator.validate(req_data, sensor_type)
+    valid = True
```

Comment:
Add a TODO for writing data to database.

Reply:
Actually it's there :-)

Reply:
Ah, I see! Sorry I need to look properly! 👓